src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -141,6 +142,7 @@ export class ArtworkApp extends React.Component<Props> {
                 <ArtworkImageBrowser artwork={artwork} />
                 <ArtworkSidebar artwork={artwork} />
                 <ArtworkDetails artwork={artwork} />
+                <PricingContext artwork={artwork} />
```

Comment:
I think we'll want to add some logic here-- the lab feature check exists in metaphysics, but here we need to verify:
- Artwork is for sale
- Artwork is _not_ in an auction

Otherwise we don't want to show the graph. I guess we could also put that in metaphysics, but might make more sense here.

Reply:
Let's do it in metaphysics. That way we don't end up fetching the histogram for works that it won't be shown for.